include/secp256k1_extrakeys.h

```diff
@@ -240,6 +240,22 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_keypair_xonly_tweak_add
     const unsigned char *tweak32
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3);
 
+/** Compare two public keys using lexicographic order of their compressed
+ *  serialization.
+ *
+ *  Returns: <0 if the first public key is less than the second
+ *           >0 if the first public key is greater than the second
+ *           0 if the two public keys are equal
+ *  Args: ctx:      a secp256k1 context object.
+ *  In:   pubkey1:  first public key to compare
+ *        pubkey2:  second public key to compare
+ */
+SECP256K1_API int secp256k1_pubkey_cmp(
+    const secp256k1_context *ctx,
+    const secp256k1_pubkey *pk1,
+    const secp256k1_pubkey *pk2
+) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3);
```

Comment:
Remind me what the difference was to the function `secp256k1_ec_pubkey_cmp`. Is there no difference and we simply overlooked this? I suspect what happened is this:
 - When we had x-only keys as input to key agg, we had to add a comparison function for x-only keys (to extrakeys module)
 - Then we switched to compressed keys as input, and we changed the comparison function to also take compressed keys, not noticing that there's one already in `secp256k1.h`

Reply:
There is no difference. I removed the commit.